sgm/inference/api.py

```diff
@@ -242,17 +291,46 @@ def image_to_image(
             samples,
             force_uc_zero_embeddings=["txt"] if not self.specs.is_legacy else [],
             return_latents=return_latents,
-            filter=None,
+            filter=filter,
         )
 
+    def wrap_discretization(
+        self, discretization, image_strength=None, noise_strength=None, steps=None
+    ):
+        if isinstance(discretization, Img2ImgDiscretizationWrapper) or isinstance(
+            discretization, Txt2NoisyDiscretizationWrapper
+        ):

```

Comment:
You should use the tuple form for `isinstance`
```suggestion
        if isinstance(discretization, (Img2ImgDiscretizationWrapper, Txt2NoisyDiscretizationWrapper)):
```
but tbh these two should then have a common superclass.